src/chocolatey/infrastructure.app/services/PowershellService.cs

```diff
@@ -435,6 +435,8 @@ public void prepare_powershell_environment(IPackage package, ChocolateyConfigura
 
             Environment.SetEnvironmentVariable("chocolateyPackageName", package.Id);
             Environment.SetEnvironmentVariable("packageName", package.Id);
+            Environment.SetEnvironmentVariable("chocolateyPackageId", package.Id);
+            Environment.SetEnvironmentVariable("packageId", package.Id);

```

Comment:
We should deprecate the non-`chocolatey` prefix environment variable names. So I don't think we should add this one in at all.